addons/im_livechat/models/discuss_channel.py

```diff
@@ -26,13 +26,25 @@ class DiscussChannel(models.Model):
     chatbot_current_step_id = fields.Many2one('chatbot.script.step', string='Chatbot Current Step')
     chatbot_message_ids = fields.One2many('chatbot.message', 'discuss_channel_id', string='Chatbot Messages')
     country_id = fields.Many2one('res.country', string="Country", help="Country of the visitor of the channel")
+    livechat_failure = fields.Selection(
+        selection=[
+            ("never_answered", "Never Answered"),
+            ("no_one_available", "No one Available"),
```

Comment:
Maybe use `no_answer` and `no_agent` as technical keys.

There is value in having them as short as possible as they need to be stored in database for each channel.